README.md

```diff
@@ -9,3 +9,39 @@ stanza-plugins contains plugins for the [Stanza Log Agent](https://github.com/ob
 All plugins in this repository are released together in a single operation.
 
 Maintainers can create a release using the GitHub UI by creating a release that adheres to Golang's [semver](https://godoc.org/github.com/rogpeppe/go-internal/semver) format. (`v1.2.3`)
+
+## Development Guidelines
+
+### Plugin Parameters
+
+#### Labels
+
+* Should be written in title case.
+
+#### Descriptions
+
+* Should be written in complete (or mostly complete) sentences and end in a period.
+* Avoid repeating the parameter label in the description. The label will already appear with the description.
+  * Container name
+    * BAD: The name of the container
+    * GOOD: Specify a single name or use * to collect logs from all containers.
+* Descriptions should offer additional information on proper configuration.
+  * Listen IP
+    * BAD: A syslog ip address of the form `<ip>`
+    * GOOD: Typically 0.0.0.0 for most configurations.
```

Comment:
Albeit this is a tad nitpicky, but this feels more like a `note` than a `description`. I'm not sure if it would be worth to apply the suggestion but I am kind of getting inspiration from other platforms that highlight descriptions of similar parameters like this 
<img width="825" alt="image" src="https://user-images.githubusercontent.com/32067685/144295632-04d5421f-57bb-42f6-a310-bfd02ce28c82.png">
 ```suggestion
    * GOOD: A network interface for the agent to bind. Typically 0.0.0.0 for most configurations.
```

Any thoughts?